internal/podman/client.go

```diff
@@ -181,7 +181,9 @@ func (o *client) Diagnostics(ctx context.Context) *Diagnostic {
 	info, err := o.machineInspect(ctx)
 	if err != nil {
 		d.MachineFound = false
-		d.Errors = append(d.Errors, fmt.Errorf("failed to detect podman machine: %w", err).Error())
+		if d.MachineRequired {
+			d.Errors = append(d.Errors, fmt.Errorf("failed to detect podman machine: %w", err).Error())
```

Comment:
```suggestion
			d.Errors = append(d.Errors, fmt.Sprintf("failed to detect podman machine: %s", err))
```
don't use an error if you don't need it